library/src/test/kotlin/build/buf/connect/protocols/ConnectInterceptorTest.kt

```diff
@@ -90,6 +90,34 @@ class ConnectInterceptorTest {
         assertThat(request.headers[CONTENT_ENCODING]).isNullOrEmpty()
         assertThat(request.headers["key"]).containsExactly("value")
         assertThat(request.contentType).isEqualTo("content_type")
+        assertThat(request.headers[USER_AGENT]).containsExactly("connect-kotlin/dev")
+    }
+
+    @Test
+    fun requestHeadersCustomUserAgent() {
+        val config = ProtocolClientConfig(
+            host = "https://buf.build",
```

Comment:
just a note that @smallsamantha will probably want to pick up these as further changes in her sweep in https://github.com/bufbuild/connect-kotlin/pull/79